### ISSUE TITLE
Dynamic Notch macOS app?

### ISSUE THREAD

**Issue by dreinert1** [Pinball3D/DynamicNotchPlugins#1]
Does a dynamic notch macOS app exist? I’ve been trying to find one. Is it called something else?

**Comment by Pinball3D** [comment]
> Does a dynamic notch macOS app exist? I’ve been trying to find one. Is it called something else?

Yeah I was working on the app for a while and this repo was supposed to allow people to submit plugins to support more things. It doesn't work very well, but I can send you the code if you want.
